web/src/test/auto/integrated/test_utils.js

```diff
@@ -82,41 +77,77 @@ export function setupKMW(kmwOptions, timeout) {
  * @param {*} functor   A callback to handle the script element.
  * @returns
  */
-export function setupScript(src, timeout, functor) {
-  return new Promise((resolve, reject) => {
+export function setupScript(src, timeout, attemptCount, existingTimer) {
+  attemptCount = attemptCount || 1;
+  if(attemptCount > 1) {
+    console.log("Re-attempting load of script '" + src + "': retry #" + attemptCount);
```

Comment:
Part of me thinks it may be worth upgrading this to a `warn`.  Assuming such statements pass through to our logging when run via BrowserStack, we might be able to have them highlighted by TC.

I'm not 100% sure that they actually _would_ pass through to our logging, though.